src/server/api/controllers/waypoints.controller.js

```diff
@@ -0,0 +1,81 @@
+const HttpError = require("../lib/utils/http-error");
+const knex = require("../../config/db");
+
+// Method for getting waypoints by id:
+const getWaypointsById = async id => {
+  try {
+    const waypoints = await knex("waypoints")
+      .select("id", "lon", "lat", "created_at", "updated_at")
+      .where({ id: id });
+    if (waypoints.length === 0) {
+      throw new HttpError(
+        "Bad request",
+        `Cannot find waypoints for ID ${id}!`,
+        409
+      );
+    }
+    return waypoints;
+  } catch (err) {
+    return err.message;
+  }
+};
+
+// Method for creating a waypoints
+
+const createWaypoints = async ({ body }) => {
+  const { lat, lon, port_id, route_id } = body;
```

Comment:
`route_id` is already renamed to `suggested_route_id` on develop. You need to pull develop and resolve merge conflicts. I recommend that you do it after my PR fixing the seeds has been merged.